qiskit_ibm_runtime/ibm_runtime_service.py

```diff
@@ -162,30 +162,27 @@ def __init__(
         account_credentials, account_preferences = self._resolve_credentials(
             token=token, locator=locator, **kwargs
         )
+        self._auth = auth
+
+        # Common attributes
+        self._backends: Dict[str, "ibm_backend.IBMBackend"] = {}
+        self._programs: Dict[str, RuntimeProgram] = {}
```

Comment:
This will also conflict with https://github.com/Qiskit/qiskit-ibm-runtime/pull/49. Can you take a look and consider merging #49 first?